utils/utils.go

```diff
@@ -21,6 +22,14 @@ var (
 
 	Kong140Version = semver.MustParse("1.4.0")
 	Kong300Version = semver.MustParse("3.0.0")
+	Kong340Version = semver.MustParse("3.4.0")
+)
+
+var ErrorConsumerGroupUpgrade = errors.New(
+	"a rate-limiting-advanced plugin with config.consumer_groups\n" +
+		"and/or config.enforce_consumer_groups was found. Please use Consumer Groups scoped\n" +
+		"Plugins when running against Kong Enterprise 3.4.0 and above.\n\n" +
+		"Check DOC_LINK for more information",
```

Comment:
Isn't `DOC_LINK` replacement with a real URL missing?

Reply:
Ah, I've just read the Slack thread where you point out it's because 3.4 docs are not live yet so there's no link to it. 👍